azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageView.kt

```diff
@@ -114,47 +116,64 @@ private fun BasicChatMessage(viewModel: MessageViewModel) {
-                                    modifier = Modifier.padding(PaddingValues(end = ChatCompositeTheme.dimensions.messageUsernamePaddingEnd))
-                                )
-                            }
-                            if (viewModel.showTime) {
-                                BasicText(
-                                    viewModel.message.createdOn?.format(timeFormat)
-                                        ?: "Unknown Time",
-                                    style = ChatCompositeTheme.typography.messageHeaderDate,
-                                )
+                Box(
+                    modifier = Modifier.padding(ChatCompositeTheme.dimensions.messagePadding)
+                ) {
+                    Column {
+                        if (viewModel.showUsername || viewModel.showTime) {
+                            Row {
+                                if (viewModel.showUsername) {
+                                    BasicText(
+                                        viewModel.message.senderDisplayName ?: "Unknown Sender",
+                                        style = ChatCompositeTheme.typography.messageHeader,
+                                        modifier = Modifier.padding(PaddingValues(end = ChatCompositeTheme.dimensions.messageUsernamePaddingEnd))
+                                    )
+                                }
+                                if (viewModel.showTime) {
+                                    BasicText(
+                                        viewModel.message.createdOn?.format(timeFormat)
+                                            ?: "Unknown Time",
+                                        style = ChatCompositeTheme.typography.messageHeaderDate,
+                                    )
+                                }
                             }
                         }
-                    }
-                    if (viewModel.message.messageType == ChatMessageType.HTML) {
-                        HtmlText(html = viewModel.message.content ?: "Empty")
-                    } else {
-                        BasicText(
-                            text = viewModel.message.content ?: "Empty"
-                        )
+                        if (viewModel.message.messageType == ChatMessageType.HTML) {
+                            HtmlText(html = viewModel.message.content ?: "Empty")
+                        } else {
+                            BasicText(
+                                text = viewModel.message.content ?: "Empty"
+                            )
+                        }
                     }
                 }
             }
+            androidx.compose.animation.AnimatedVisibility(visible = viewModel.isRead) {
+                Icon(
+                    painter =
+                    painterResource(
+                        id =
+                        R.drawable.azure_communication_ui_chat_ic_fluent_message_read_10_filled
+                    ),
```

Comment:
tent is required to display right color for icon

Reply:
addressed